osidb/serializer.py

```diff
@@ -890,6 +972,7 @@ class Meta:
             + ACLMixinSerializer.Meta.fields
             + AlertMixinSerializer.Meta.fields
             + TrackingMixinSerializer.Meta.fields
+            + HistoryMixinSerializer.Meta.fields
```

Comment:
nitpick: I am sure that it is not always possible but when it is I would prefer ordering the list-like structures alphabetically so a human can easily see what is there and what is not